src/components/meeting/JitsiComponentWrapper.js

```diff
@@ -2,37 +2,54 @@ import {JitsiComponent} from "e-stella-jitsi";
 import {MeetingFab} from "./MeetingFab";
 import {useState} from "react";
 import {Whiteboard} from "e-stella-whiteboard";
+import {TaskWrapper} from "../task/TaskWrapper";
 
 export const JitsiWrapper = (props) => {
 
     const [jitsiHeight, setJitsiHeight] = useState('calc(100vh - 60px)')
     const [jitsiWidth, setJitsiWidth] = useState('100%')
-    const [whiteboardIsSeen, setWhiteboardIsSeen] = useState(false)
+    const [componentSeen, setComponentSeen] = useState(null)
 
     const addWhiteboard = () => {
         setJitsiWidth('20%')
         setJitsiHeight('calc(100vh - 60px)')
-        setWhiteboardIsSeen(true)
+        setComponentSeen(
+            <div style={{float: "left", width: '80%', height: 'calc(100vh-60px)'}}>
+                <Whiteboard
+                    client_id={process.env.REACT_APP_WHITEBOARD_KEY}
+                    code={getWhiteboardCode()}
+                />
+            </div>
+        )
     }
 
     const defaultView = () => {
         setJitsiHeight('calc(100vh-60px)')
         setJitsiWidth('100%')
-        setWhiteboardIsSeen(false)
+        setComponentSeen(null)
+    }
+
+    const showTask = () => {
+        setJitsiWidth('20%')
+        setComponentSeen(
+            <div style={{float: "left", width: '80%', height: 'calc(100vh-60px)'}}>
+                <TaskWrapper id={16} submitLeftOffset={'25%'}/>
+            </div>
+        )
     }
 
     const getWhiteboardCode = () => {
-        if(props.roomName.length < 15) return 'abcdefghijklmnopqrst'
+        if (props.roomName.length < 15) return 'abcdefghijklmnopqrst'
```

Comment:
Tu ten roomName też nie powinien być jakoś powiązany z rozmową czy czymś bo jesli dobrze rozumiem to teraz wszystkie rozmowy naraz będą korzystać z tego samego whiteboard?

Reply:
roomName z tego, co pamiętam, to uuid interview. Ten if był na wypadek, gdyby uuid było krótsze niż 15 znaków, czyli wtedy, kiedy robiliśmy pokazówkę XD